source/application.py

```diff
@@ -0,0 +1,18 @@
+from flask import Flask
+from werkzeug.contrib.fixers import ProxyFix
+
+from endpoints.api import api
+from model.Database import Database
+from plugins.simpleLock.simpleLock import simpleLock
+
+
+DAO = Database()
```

Comment:
Why not call it, for example, device_database? According to proper [PIP8](https://www.python.org/dev/peps/pep-0008/) conventions, variable names should be lowercase. I suggest that the server code adheres to this as well, as most if not all libraries you use will do the same (creating consistency in the code). Should be very easy to do if you are using the right IDE!